### PR TITLE
Adds empty merged input object type to composition

### PR DESCRIPTION
**Pull request by PascalSenn**

### CODE REVIEW

File: spec/Section 4 -- Composition.md [hunk 1]
```diff
@@ -18,4 +18,49 @@ run in sequence to produce the composite execution schema.
...
+**Error Code**
+
+INPUT_OBJECT_TYPE_EMPTY
+
+**Formal Specification**
+
+- Let {inputs} be the set of all input object in the merged schema
+- For each {input} in {inputs}:
+  - {IsInputObjectTypeEmpty(input)} must be false
+
```
Comment (PascalSenn) [inline spec/Section 4 -- Composition.md:32]: TODO: Exlude required fields

File: spec/Section 4 -- Composition.md [hunk 2]
```diff
@@ -18,4 +18,49 @@ run in sequence to produce the composite execution schema.
...
 
+#### Empty Merged Input Object Type
+
+**Error Code**
+
+INPUT_OBJECT_TYPE_EMPTY
+
+**Formal Specification**
+
+- Let {inputs} be the set of all input object in the merged schema
```
Comment (glen-84) [inline spec/Section 4 -- Composition.md:29]:
```suggestion
- Let {inputs} be the set of all input objects in the merged schema
```

File: spec/Section 4 -- Composition.md [hunk 3]
```diff
@@ -18,4 +18,49 @@ run in sequence to produce the composite execution schema.
...
+```graphql
+input Input1 {
+  field1: String @inaccessible
+  field2: Int
+}
+```
+
+This counter-example shows an invalid merged input object type. 
+The `Input1` type is defined in two source schemas, but both `field1` and `field2` are flagged as
+`@inaccessible` in at least on of the schemas:
```
Comment (glen-84) [inline spec/Section 4 -- Composition.md:57]:
```suggestion
`@inaccessible` in at least one of the schemas:
```

**Review by glen-84 — CHANGES_REQUESTED** [review]

**Comment by PascalSenn** [comment]
closed by #93

**Pull request closed without merging**